src/main/webapp/app/overview/course-lectures/course-lecture-details.component.ts

```diff
@@ -86,6 +88,14 @@ export class CourseLectureDetailsComponent implements OnInit {
             });
     }
 
+    showGoToLectureManagementButton(): boolean {
+        return !!this.lecture?.course?.isAtLeastTutor;
+    }
+
+    redirectToLectureManagement(): void {
+        this.router.navigate([`/course-management/${this.lecture?.course?.id}/lectures/${this.lecture?.id}`]);
```

Comment:
```suggestion
        this.router.navigate(["/course-management", this.lecture?.course?.id, "lectures", this.lecture?.id]);
```
I think the array for `navigate` is supposed to include the different parts of the path to navigate to.

Reply:
Done